### PR TITLE
Save rating issue decision date on request_issue

### PR DESCRIPTION
**Pull request by pkarman**
connects #10070 

## Description

Save the original rating issue decision date on the request issue so we do not rely on the presence of the rating issue in the future to show the decision date.

### CODE REVIEW

File: client/app/intake/util/issues.js [hunk 1]
```diff
@@ -202,6 +202,7 @@ const formatRatingRequestIssues = (state) => {
       return {
         request_issue_id: issue.id,
         rating_issue_reference_id: issue.ratingIssueReferenceId,
+        rating_issue_decision_date: issue.decisionDate,
```
Comment (leikkisa) [inline client/app/intake/util/issues.js]: Does this need the same formatting as the nonrating issue?
formatDateStringForApi

Comment (leikkisa) [reply client/app/intake/util/issues.js]: Should we just call this decision_date here also?

File: app/models/request_issue.rb [hunk 2]
```diff
@@ -297,6 +298,9 @@ def approx_decision_date_of_issue_being_contested
       decision_date
     elsif decision_issues.any?
       decision_issues.first.approx_decision_date
```
Comment (leikkisa) [inline app/models/request_issue.rb]: This isn't about your PR but the name of this method seems ambiguous, given that in the top two cases it results in the decision_date of the contested issue, and in the last situation it results in the decision_date of the request issue.

Comment (pkarman) [reply app/models/request_issue.rb]: I think 99% of the time it's going to be one of the first two cases.

The only way it gets to the 3rd case is if the request issue somehow represents the contention of a decision w/o a rating. I.e. the first link in the chain.

Comment (pkarman) [reply app/models/request_issue.rb]: we could try removing it and seeing what tests break...

Comment (pkarman) [reply app/models/request_issue.rb]: nothing broke so I removed that last condition.

**Pull request merged**